repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/derp/DerpResponseMessagingTest.groovy

```diff
@@ -19,10 +19,12 @@
  */
 package features.filters.derp
 
+import org.junit.Ignore
 import org.openrepose.framework.test.ReposeValveTest
 import org.rackspace.deproxy.Deproxy
 import org.rackspace.deproxy.MessageChain
 
+@Ignore
```

Comment:
I thought we were going to using the Scripting filter to do what RMS did for the sake of the functional tests?

Reply:
This test is honestly more of a RMS test than a DeRP test. That is, this test verifies that RMS can process the response written by DeRP. Since RMS is not currently supported, that is not the case, and so I don't believe it makes sense to replace RMS with the Scripting Filter in this case.

I should note that we have other tests to verify the behavior of the DeRP filter. While there are more test cases I can think to add, I think writing new tests is outside the scope of this story.